src/components/component.ts

```diff
@@ -246,10 +243,10 @@ module Plottable {
     }
 
     /**
-     * Renders the Component into a given DOM element. The element must be as <svg>.
+     * Renders the Component to a given DOM element. The element must be an <svg>.
```

Comment:
Can you put the second sentence on a different line?  Unless you think that doesn't make sense